### PR TITLE
[Lite] ICU rebasing bug fix.

### PR DESCRIPTION
**Pull request by heke123**
build pass both enable/disable use_icu_alternatives_on_android.
Bug=xwalk-4520

### CODE REVIEW

**Comment by heke123** [comment]
@halton please help to review.thanks.

**Comment by halton** [comment]
@heke123 could you include someone in team? I'd like to build the pipeline instead of all depends on my review.

**Comment by heke123** [comment]
@d0ngjun please help to review, thanks.

File: ui/base/l10n/l10n_util.cc [hunk 1]
```diff
@@ -24,6 +24,13 @@
 #include "base/strings/sys_string_conversions.h"
 #include "base/strings/utf_string_conversions.h"
 #include "build/build_config.h"
+#if defined(OS_ANDROID)
```
Comment (d0ngjun) [inline ui/base/l10n/l10n_util.cc]: seems this code change is unnecessary.

File: ui/base/l10n/l10n_util_android.cc [hunk 2]
```diff
@@ -83,6 +83,13 @@ ScopedJavaLocalRef<jobject> NewJavaLocale(
 
 }  // namespace
 
+#if defined(USE_ICU_ALTERNATIVES_ON_ANDROID)
+//Fix:Compliler complains Java_Locali_getJavaLocale defined but not used.
```
Comment (d0ngjun) [inline ui/base/l10n/l10n_util_android.cc]: use NOTE better. Fix could be thought as an issue.

**Comment by heke123** [comment]
@d0ngjun please help to review, thanks.

**Comment by d0ngjun** [comment]
LGTM

**Comment by halton** [comment]
Will merge, thanks @d0ngjun and @heke123

**Pull request merged**